src/commands/format.js

```diff
@@ -38,7 +39,12 @@ async function formatCode(interaction) {
 
   const msg = messages.first();
   const code = msg.content;
-  interaction.channel.messages.delete(msg.id);
+
+  interaction.channel.messages.delete(msg.id).catch((error) => {
+    if (error.code !== Constants.APIErrors.UNKNOWN_MESSAGE) {
+      console.error('Failed to delete the message:', error);
+    }
```

Comment:
So this ignores the error if it's the `UNKNOWN_MESSAGE` error?

Reply:
Yes, that is correct. I just did it as demonstrated in the docs. The bug was caused by trying to delete an already deleted msg.